desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -773,9 +776,20 @@ private JComponent getWindowPanelContent() {
         final NewsChannelStatusLabel newChannelStatusLabel = new NewsChannelStatusLabel(rightInformationPanel, _userPreferences);
         statusBar.add(newChannelStatusLabel);
         statusBar.add(Box.createHorizontalStrut(20));
-
-        final LicenceAndEditionStatusLabel statusLabel = new LicenceAndEditionStatusLabel(rightInformationPanel);
-        statusBar.add(statusLabel);
+        
+        if (Version.isCommunityEdition()) {
+            final CommunityEditionStatusLabel statusLabel = new CommunityEditionStatusLabel(rightInformationPanel);
+            statusBar.add(statusLabel);
+            statusBar.add(Box.createHorizontalStrut(20));
+        } else {
+            final ServiceLoader<PlugabblePanel> pluggablePanelsLoaders = ServiceLoader.load(PlugabblePanel.class);
```

Comment:
Hmmmm... Something seems a bit off here. You should probably make a factory/descriptor for the icon and panel in the factory, like:
```
interface InformationPanelDescriptor {
    JComponent createInformationComponent(AnalysisJobBuilderWindow analysisJobBuilderWindow);
    String getLabelImagePath();
    String getLabelText();
}
```

... And load that using the service loader instead. That also removes the dependency on DCPanel, any JComponent is (and should be) usable with the RightInformationPanel.

I'd maybe also drop the PluggableRightPanelLabel name... The label itself is not pluggable, and the position can change (yes, I know RightInformationPanel is already misnamed in that regard, but let's not make it worse :)). How about just InformationPanelLabel?